google/cloud/bigtable/completion_queue_test.cc

```diff
@@ -121,42 +126,47 @@ class MockClientAsyncReaderInterface
   MOCK_METHOD2(Finish, void(grpc::Status*, void*));
   MOCK_METHOD2_T(Read, void(Response*, void*));
 };
+
 /// @test Verify that completion queues can create async operations.
 TEST(CompletionQueueTest, AyncRpcSimple) {
   MockClient client;
 
-  auto reader =
-      google::cloud::internal::make_unique<testing::MockAsyncApplyReader>();
+  using ReaderType =
+      ::google::cloud::bigtable::testing::MockAsyncResponseReader<
+          btadmin::Table>;
+  auto reader = google::cloud::internal::make_unique<ReaderType>();
   EXPECT_CALL(*reader, Finish(_, _, _))
-      .WillOnce(
-          Invoke([](btproto::MutateRowResponse*, grpc::Status* status, void*) {
-            *status = grpc::Status(grpc::StatusCode::OK, "mocked-status");
-          }));
+      .WillOnce(Invoke([](btadmin::Table* table, grpc::Status* status, void*) {
+        table->set_name("fake/table/name/response");
+        *status = grpc::Status(grpc::StatusCode::OK, "mocked-status");
+      }));
 
-  EXPECT_CALL(client, AsyncMutateRow(_, _, _))
+  EXPECT_CALL(client, AsyncGetTable(_, _, _))
       .WillOnce(Invoke([&reader](grpc::ClientContext*,
-                                 btproto::MutateRowRequest const&,
+                                 btadmin::GetTableRequest const& request,
                                  grpc::CompletionQueue*) {
+        EXPECT_EQ("fake/table/name/request", request.name());
         return std::unique_ptr<grpc::ClientAsyncResponseReaderInterface<
             // This is safe, see comments in MockAsyncResponseReader.
-            btproto::MutateRowResponse>>(reader.get());
+            btadmin::Table>>(reader.get());
       }));
 
   auto impl = std::make_shared<testing::MockCompletionQueue>();
   bigtable::CompletionQueue cq(impl);
 
   // In this unit test we do not need to initialize the request parameter.
-  btproto::MutateRowRequest request;
+  btadmin::GetTableRequest request;
+  request.set_name("fake/table/name/request");
```

Comment:
Is the comment on line 157 (and 206) now misleading?

Reply:
Fixed.

Reply:
I think I see the 206 update but not the 157 one.